netbox_data_import/templates/netbox_data_import/import_preview.html

```
@@ -496,6 +496,22 @@ <h5 class="mb-0"><i class="mdi mdi-eye"></i> Import Preview — Step 2 of 3</h5>
 
                 {% elif row.action == 'error' and row.object_type == 'device' and profile_id %}
                   {# Device error — show class mapping button if we have source_class #}
+                  {% if row.extra_data.identity_conflict == 'duplicate_name' or row.extra_data.identity_conflict == 'name_placement_conflict' %}
+                  {% if row.extra_data.suggested_name and row.source_id %}
+                  <form method="post" action="{% url 'plugins:netbox_data_import:resolve_duplicate_name' %}" class="d-inline-flex align-items-center gap-1">
+                    {% csrf_token %}
+                    <input type="hidden" name="profile_id" value="{{ profile_id }}">
+                    <input type="hidden" name="source_id" value="{{ row.source_id }}">
+                    <input type="hidden" name="row_number" value="{{ row.row_number }}">
+                    <input type="text" name="new_name" value="{{ row.extra_data.suggested_name }}"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)